### ISSUE TITLE
번역 : Intro to property based tests

### ISSUE THREAD

**Issue by MiryangJung** [MiryangJung/learn-go-with-tests-ko#16]
https://github.com/MiryangJung/learn-go-with-tests-ko/blob/master/ko-KR/roman-numerals.md

**Comment by 14hhan** [comment]
제가 하겠습니다!

**Comment by MiryangJung** [comment]
네 감사합니다! [번역 규칙](https://github.com/MiryangJung/learn-go-with-tests-ko#%EB%B2%88%EC%97%AD-%EA%B8%B0%EC%97%AC) 참고해주세요!
